sdcm/nemesis.py

```diff
@@ -180,6 +180,23 @@ def disrupt_hard_reboot_node(self):
         self.log.info('Waiting JMX services to start after node reboot')
         self.target_node.wait_jmx_up()
 
+    def disrupt_multiple_hard_reboot_node(self):
+
+        num_of_reboots = random.randint(2, 10)
+        for i in range(num_of_reboots):
+            self._set_current_disruption('MultipleHardRebootNode %s' % self.target_node)
+            self.target_node.reboot(hard=True)
+            if random.choice([True,False]):
```

Comment:
what is the logic here?

Reply:
randomly go to two different path. One is only waiting for db up, another is waiting for jvm up.

@yarongilor is it right?

Reply:
That's right. using randomization for the purpose of wider coverage/scenarios